pkgs/development/python-modules/tidalapi/default.nix

```
@@ -0,0 +1,40 @@
+{ lib
+, stdenv
+, buildPythonPackage
+, fetchPypi
+, python-dateutil
+, requests
+, ffmpeg-python
+, keyring
+, pillow
+, pytest
```

Use `pytestCheckHook` instead.

```suggestion
, pytestCheckHook
```

---

I tried to use it, but the tests failed because they try to connect to the internet, which is forbidden in the sandbox.

---

Disable the individual tests using `disabledTests`.

---

> Disable the individual tests using `disabledTests`.

I analyzed test by test and all require internet access

---

Then add a comment to `doCheck` that says so.